dockerfile_functions.sh

```diff
@@ -651,6 +681,12 @@ RUN set -eux; \\
     case "\${ARCH}" in \\
 EOI
 	print_java_install_pre "${file}" "${pkg}" "${bld}" "${btype}" "${osfamily}" "${os}"
+	if [ "${btype}" == "slim" && ${os} == "ubi"]; then
```

Comment:
Formatting should be fixed
```
if [ "${btype}" == "slim" && "${os}" == "ubi" ];
```
You can combine the two statements for ubi and ubi-minimal
```
if [ "${btype}" == "slim" ]
    if [ "${os}" == "ubi" ];
        ...
    elif [ "${os}" == "ubi-minimal" ];
        ...
```

Reply:
Combined statements as recommended